relayer/relay/src/error.rs

```diff
@@ -28,6 +28,17 @@ pub enum Kind {
     /// Trusted store error, raised by instances of `Store`
     #[error("store error")]
     Store,
+
+    /// Event error (raised by the event monitor)
+    #[error("Bad Notification")]
+    Event,
+
+    /// Response parsing error
+    #[error("Could not parse/unmarshall response")]
+    ResponseParsing,
+
```

Comment:
Nitpick but still important: all other error types (above `ResponseParsing`) have a comment. Can we have one here as well?

Reply:
Good find, fixed.